exercises/en/test_03_10.py

```diff
@@ -0,0 +1,13 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+
+    # If an assertion fails, the message will be displayed
+
+    assert ".pivot" in __solution__, "Make sure you are using the pivot function."
```

Comment:
Can you test the column labels and do some tests for the column sums? i would recommend that the first test always be if they named the testable object correctly